python/cudf/cudf/pandas/__init__.py

```diff
@@ -50,7 +50,7 @@ def install():
             mr = rmm.mr.CudaAsyncMemoryResource(initial_pool_size=free_memory)
             rmm.mr.set_current_device_resource(mr)
         elif rmm_mode == "managed":
-            mr = rmm.mr.ManagedMemoryResource()
+            mr = rmm.mr.PrefetchResourceAdaptor(rmm.mr.ManagedMemoryResource())
```

Comment:
Why is only “managed” prefetching? I think we were testing a prefetching managed _pool_. Worth checking before we merge.

Reply:
@GregoryKimball @vyasr Can you recommend on which all rmm modes need to have prefetching enabled?

Reply:
Same thoughts as Bradley about the `managed_pool` having prefetch on sub-allocate.

Reply:
I spoke with @galipremsagar and will pursue this fix.

Reply:
Done in 7047b77366c93723de8a3c40fd3a2fbb16c05c19.